tests/test_a_telegram_asyncio.py

```diff
@@ -82,13 +82,15 @@ def gen_callback(text):
 class TestChatModel:
 
     @staticmethod
+    @pytest.mark.a_telegram_asyncio
```

Comment:
Why have you added all these marks? We can just run `pytest tests/filename.py` if we want to ^^"